modules/core/client/directives/pagination.client.directive.html

```diff
@@ -0,0 +1,7 @@
+<div class="list-group">
+    <div data-ng-repeat="item in pagedItems">
```

Comment:
I don't think I can have a track by with this directive unless we make an option/attribute to pass in what to track by.  Since this can be used to paginate/search any array we really can't guarantee that a object._id is going to be available.  What would you suggest tracking by? $index?